addons/main/functions/fnc_showDamageFeedbackMarker.sqf

```diff
@@ -11,14 +11,16 @@ private _selfOrUnkownDamage = isNull _instigator || {_unit isEqualTo _instigator
 private _ctrl = _display ctrlCreate ["RscPictureKeepAspect", -1];
 _ctrl ctrlSetBackgroundColor [0, 0, 0, 1];
 _ctrl ctrlSetPosition [0, 0, 1, 1];
```

Comment:
TBH you do not need more files for the damage markers, you just need to change this and make that smaller or bigger. Then you can just use a slider CBA setting.